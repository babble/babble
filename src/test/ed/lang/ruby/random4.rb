
a = [ 1 , nil , 3 ]
puts a.length
a = a.compact
puts a.length
