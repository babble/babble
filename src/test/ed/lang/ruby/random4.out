3
2
adkjad1ad
